pyproject.toml

```diff
@@ -15,7 +15,7 @@ packages = [{include = "warp_journal"}]
 warp-journal = 'warp_journal.__main__:main'
 
 [tool.poetry.dependencies]
-python = "^3.8.1"
+python = "^3.12.5"
```

Comment:
I'm not willing to pin this to the latest version because Windows users with the bundled .exe are not the only users of this tool. As long as Python 3.8 (or 3.9) still works here, we should allow it.

Also, besides changing the minium version to 3.9 it would be much simpler to change the single line that requires it.